mlir/include/Catalyst/Transforms/Passes.td

```diff
@@ -17,6 +17,12 @@
 
 include "mlir/Pass/PassBase.td"
 
+def DetensorizeSCFPass : Pass<"detensorize-scf"> {
+    let summary = "Detensorize for, if, while operations from the SCF dialect.";
+
```

Comment:
One needs to declare which dialects the pass needs. Since you are creating tensor and scf-ops, we need to load them.

```suggestion

let dependentDialects = [
        "tensor::tensorDialect",
        "scf::scfDialect",
    ];
```

Reply:
Good catch! SCF is not required because it is necessarily in the input.
1353d7f07cdbdd5e0ac8f110bbb2fd2c13aba4e5